src/Tempest/View/src/Renderers/TempestViewCompiler.php

```diff
@@ -131,7 +131,7 @@ private function applyAttributes(array $elements): array
                 ->setChildren($children);
 
             foreach ($element->getAttributes() as $name => $value) {
-                $attribute = $this->attributeFactory->make($name);
+                $attribute = $this->attributeFactory->make($element, $name);
 
                 $element = $attribute->apply($element);
```

Comment:
It would make more sense to change the `apply` method to also get the attributeName

Reply:
Or maybe not